src/avatar-group.scss

```diff
@@ -81,6 +81,17 @@ $block: #{$fd-namespace}-avatar-group;
     }
   }
 
+  &__overflow--body {
```

Comment:
should be `&__overflow-body`

Reply:
`__overflow--body` means that there's a class `__overflow` and `--body` is modifying this class

Reply:
@InnaAtanasova fixed